frontend/lib/item_listing/view_models/filter.viewmodel.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:utm_marketplace/item_listing/model/filter_options.model.dart';
+import 'package:utm_marketplace/item_listing/view_models/listing.viewmodel.dart';
+import 'package:utm_marketplace/shared/view_models/loading.viewmodel.dart';
+
+class FilterViewModel extends LoadingViewModel {
+  SortOrder? _sortOrder;
+  DateTime? _dateFrom;
+  String? _condition;
+  double? _minPrice;
+  double? _maxPrice;
+  String? _campus;
+
+  SortOrder? get sortOrder => _sortOrder;
+  DateTime? get dateFrom => _dateFrom;
+  String? get condition => _condition;
+  double? get minPrice => _minPrice;
+  double? get maxPrice => _maxPrice;
```

Comment:
You don't have to refactor this since this is more of just curiosity: any reason why you chose not to use the FilterOptions model you created?